alvr/client_openxr/src/stream.rs

```diff
@@ -457,22 +457,42 @@ fn stream_input_loop(
             return;
         }
 
-        let Some(xr_now) = crate::xr_runtime_now(xr_session.instance()) else {
+        let Some(now) = crate::xr_runtime_now(xr_session.instance()).map(crate::from_xr_time)
+        else {
             error!("Cannot poll tracking: invalid time");
             return;
         };
 
+        // All Pico headsets seem to have a problem with velocity values, to different degrees.
+        // calculating velocitied by differentiation yields jittery results. In the following
```

Comment:
Capitalization at the start of the sentence, *velocities (sorry for the nitpick)